specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.Media/MediaServices.tsp

```diff
@@ -153,7 +144,7 @@ namespace Microsoft.EventGrid.SystemEvents {
     `@odataType`: string;
```

Comment:
I believe the `@discriminator` field here should be `@odata.type` (note the added '.'):

Based on the events they have listed here: 

https://learn.microsoft.com/en-us/azure/event-grid/event-schema-media-services?tabs=cloud-event-schema#joboutputstatechange

```json
   "@odata.type": "#Microsoft.Media.JobOutputAsset",
```